### 1) Calculate the probability of flipping a balanced coin four times and getting each pattern: HTTH, HHHH and TTHH.

In [1]:
.5**4

0.0625

In [2]:
1/(2**4)

0.0625

The probability of getting each pattern is 6.25% (.5^4 = .0625). Each flip has a 50% chance of heads and a 50% chance of tails so the likelihood of any particular permutation after four flips is 50% \* 50% \* 50% \* 50% or .5^4.

When flipping a balanced coin x number of times, each permutation of heads and tails is going to be equally likely so another way of calculating this would be 1/(2^4) = 1/16 = .0625. There are 16 possible permutations and all are equally likely, so 1/16 is the probality of each particular result.

### 2) If a list of people has 24 women and 21 men, then the probability of choosing a man from the list is 21/45. What is the probability of not choosing a man?

In [3]:
(45-21)/45

0.5333333333333333

The probability of not choosing a man is 24 out of 45 or 53.333%.

### 3) The probability that Bernice will travel by plane sometime in the next year is 10%. The probability of a plane crash at any time is .005%. What is the probability that Bernice will be in a plane crash sometime in the next year?

The question does not provide enough information to answer.

It tells us that the probability Bernice will travel by plane sometime in the next year is 10%, but it does not say how many plane rides she will take or how long her plane ride(s) will be. 

It tells us that the the probability of a plane crash at any time is .005%, but is that .005% every second, minute, hour.... year? Or could it be .005% every plane ride? Does that .005% even represent a single planes probability of crashing or does it represent the probability of a single crash out of all planes? It is not clear from the question.

If we assume 10% is the probability that Bernice will take exactly one plane ride in the next year, 90% is the probability that she will take exactly zero plane rides, and .005% is the probability that a single plane ride will result in a crash, then the answer is .00005 \* .1 = .000005 or .0005%.


In [4]:
(.00005*.1)*100

0.0005

In [5]:
10*.005

0.05

### 4) A data scientist wants to study the behavior of users on the company website. Each time a user clicks on a link on the website, there is a 5% chance that the user will be asked to complete a short survey about their behavior on the website. The data scientist uses the survey data to conclude that, on average, users spend 15 minutes surfing the company website before moving on to other things. What is wrong with this conclusion?

Only users that click on a link receive a survey, so the survey is neglecting users who do not click on a link. The data scientist may be able to say that users who click on at least one link spend , on average, 15 minutes surfing the company website, but the 15 minutes should not be used as a representative average for all user.